integration-tests/smallrye-opentracing/src/main/resources/application.properties

```diff
@@ -3,5 +3,6 @@ quarkus.datasource.jdbc.url=jdbc:tracing:postgresql://localhost:5432/mydatabase
 quarkus.datasource.jdbc.driver=io.opentracing.contrib.jdbc.TracingDriver
 quarkus.datasource.username=sa
 quarkus.datasource.password=sa
+quarkus.rest.single-default-produces=false
```

Comment:
This is needed because RESTEasy Reactive will default to `text/plain` when returning `String` from a JAX-RS Resource method, but this default is not what this integration uses